Formula/postgresql@14.rb

```diff
@@ -145,8 +145,10 @@ def caveats
 
         You can migrate to a versioned data directory by running:
           mv -v "#{old_postgres_data_dir}" "#{var/name}"
+          sed -i.bak "s|<string>#{old_postgres_data_dir}</string>|<string>#{var/name}</string>|" #{opt_prefix}/homebrew.mxcl.postgresql@14.plist
+          sed -i.bak "s|-D #{old_postgres_data_dir}$|-D #{var/name}|" #{opt_prefix}/homebrew.postgresql@14.service
```

Comment:
Probably want to scope this per-OS to avoid confusion (and handle the fact that BSD and GNU `sed` work differently with `-i`, IIRC)

Reply:
They do, but actually specifying an extension (e.g. `.bak`) makes them behave the same way.

Reply:
They work differently, `-i` on Linux  works the same as `-i ''` on macOS, but `-i.bak` works the same way on both systems

Reply:
Agreed to scoping per-os if we need to have the user call `sed`, but a solution where they don't is preferable.